src/aqua/decider/register_worker.aqua

```diff
@@ -18,7 +18,10 @@ service JsonWorkerTxInfo("json"):
 func store_tx(spell_id: SpellId, tx_info: WorkerTxInfo):
     list_push(spell_id, TXS_KEY, tx_info)
 
-
+func remove_tx(spell_id: SpellId, tx_info: WorkerTxInfo):
```

Comment:
is it used anywhere?

Reply:
No, I want to use it in the future, so I left it not to forget to fix txs tracking. We may now rework the tx tracking a bit since we have `list_remove`. I guess, I add linear task for it